code/modules/mob/living/simple_animal/hostile/ordeal/steel.dm

```diff
@@ -230,20 +220,24 @@
 	deathmessage = "mutters something under their breath before collapsing."
 	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID|MOB_BUG
 	ranged_cooldown_time = 15 SECONDS
+	move_to_delay = 3
```

Comment:
Why are they so slow?

Reply:
That's same speed as sweepers, if not slightly faster.